label_studio/io_storages/migrations/0019_aperturedb.py

```diff
@@ -0,0 +1,156 @@
+# Generated by Django 3.2.23 on 2023-12-20 20:16
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Do we need to maintain a migration if our implementation isn't public yet? Or is a client using this already?

Reply:
I'm not sure the migration stuff is doing us any good ATM.